### ISSUE TITLE
Specifying the mount option when partitioning a disk

### ISSUE THREAD

**Issue by saber716rus** [calamares/calamares#1753]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
A mount point is specified, but no mount option is specified. You need a line where you can enter the mount option.

**Describe alternatives you've considered**
Depending on the fs, you can select using checkboxes and lines if there is a variable in the mount option.

**Additional context**
In principle, I briefly described everything.

**Comment by killajoe** [comment]
if you click into the mount point field you can type in custom mountpoint.
Or did i understand you wrong?

**Comment by saber716rus** [comment]
> if you click into the mount point field you can type in custom mountpoint.
> Or did i understand you wrong?

Hello. No, it's not like that. There is a mount point (/boot, /boot/efi, and so on), and there is a mount option, that is, there are mount parameters, for example, the btrfs file system, its parameters include compression or auto-fragmentation.

**Comment by demmm** [comment]
Looks like you'd like to see the distro settings for mount options to be configurable by the user.
https://github.com/calamares/calamares/blob/calamares/src/modules/fstab/fstab.conf

**Comment by saber716rus** [comment]
> Looks like you'd like to see the distro settings for mount options to be configurable by the user.
> https://github.com/calamares/calamares/blob/calamares/src/modules/fstab/fstab.conf

Yes, that's right, but not in a file, but when a partition is specified, along with the mount point, fs, and so on.

**Comment by demmm** [comment]
That is why the file was linked, it is set by distros, you now request a GUI fr users.

**Comment by saber716rus** [comment]
> That is why the file was linked, it is set by distros, you now request a GUI fr users.

Yes, that's right, sometimes you need to manually adjust for equipment, certain mount options.

**Comment by saber716rus** [comment]
Added fs label, but the mount option is not visible yet when partitioning the disk.

**Comment by emanuc** [comment]
The case of use could be to mount the partitions at the time of installation with BTRFS filesystem with a higher compression, to save space, at rebooting to have different mounting options compared to the time of installation.
Example:
Installation > mount option > **compress-force=zstd:15**
Restart > mount option > **compress=zstd:1**
I tried to search for how to do during a test up in Manjaro installation but I didn't succeed ([ISSUE](https://github.com/calamares/calamares-manjaro/issues/42)), I can easily do it with the Debian or Ubuntu installer.

**Comment by saber716rus** [comment]
> The case of use could be to mount the partitions at the time of installation with BTRFS filesystem with a higher compression, to save space, at rebooting to have different mounting options compared to the time of installation. Example: Installation > mount option > **compress-force=zstd:15** Restart > mount option > **compress=zstd:1** I tried to search for how to do during a test up in Manjaro installation but I didn't succeed ([ISSUE](https://github.com/calamares/calamares-manjaro/issues/42)), I can easily do it with the Debian or Ubuntu installer.

I have already asked this question to the maintainers of manjaro, but I was sent here.
